### ISSUE TITLE
(suggestion) add support to the Android (AArch64) and also for the PS Vita port

### ISSUE THREAD

**Issue by Thetouchedjoe** [trzy/Supermodel#110]
I'm kinda give you an suggestion to port this emulator, Supermodel to be have an Android port (only for 64 bit devices) and also a port for PS Vita (maybe will be helped by Rinneganamante I guess?). The possible version to release looks like =
- Android (64 bit devices only) =
- there's an button which is installing each game to the home screen (app installer button)
- if the games installed, there's an box art for the each arcade games with an title in the down of the box art (the arcade icon)
- when it started, there's an Arcade stick that used for aiming Crosshair and moving character and also to navigate the car with a 6 button layout used for fighting games, racing games and for Lightgun games (onscreen buttons)
- to browse out the settings, example use to Screen Scale for Normal to each size and much everything else (settings gui)

And for the PS Vita maybe it overall the same looks with the Android but it differentiated with the file manager folders and also mapped controls using the Vita buttons

I guess that for all, don't forgot to give me the response!

**Comment by dukeeeey** [comment]
I've been wondering about porting the renderer to opengl ES, it might be as simple as simply changing the context version and shader version.

**Comment by Thetouchedjoe** [comment]
Yeah dukeeey, I think that's a good idea. We're using the OpenGL ES renderer not using Vulkan. And also the onscreen buttons is not just the Analog Stick and 6 button layout. Theres a start button, Service and Test button, those button can be customizable

**Comment by tails101** [comment]
it is in retropie,batocera and recalbox on arm 64 so i hope we will get an android version soon.

**Comment by cerroverb** [comment]
this would be incredible to have as im pretty sure there is no viable model 3 emulation on android
